src/ast/mod.rs

```diff
@@ -3744,7 +3750,7 @@ pub enum Statement {
         or_alter: bool,
         name: ObjectName,
         params: Option<Vec<ProcedureParam>>,
-        body: Vec<Statement>,
+        body: BeginEndStatements,
```

Comment:
It it an abuse of BeginEndStatements to potentially have the begin/end token be empty? This could alternately be an enum of BeginEndStatements and Sequence, similar to elsewhere

Reply:
@iffyio based on the recent discussion on the other PR, guessing you will prefer this also to be an enum, to avoid the empty begin/end tokens?